src/components/fields/select-field/README.md

```diff
@@ -0,0 +1,83 @@
+| `backspaceRemovesValue` | `bool`             |    -     | -                                  | `true`  | Remove the currently focused option when the user presses backspace                                                                                                                                                                                                                         |
+| `components`            | `objectOf(func)`   |    -     | -                                  | -       | Map of components to overwrite the default ones, see [what components you can override](https://react-select.com/components)                                                                                                                                                                |
+| `filterOption`          | `func`             |    -     | -                                  | -       | Custom method to filter whether an option should be displayed in the menu                                                                                                                                                                                                                   |
+| `inputId`               | `string`           |    -     | -                                  | -       | The id of the search input                                                                                                                                                                                                                                                                  |
+| `isClearable`           | `bool`             |    -     | -                                  | -       | Is the select value clearable                                                                                                                                                                                                                                                               |
+| `isDisabled`            | `bool`             |    -     | -                                  | -       | Is the select disabled                                                                                                                                                                                                                                                                      |
+| `isOptionDisabled`      | `func`             |    -     | -                                  | -       | Override the built-in logic to detect whether an option is disabled                                                                                                                                                                                                                         |
```

Comment:
can we add this in the bottom with an explanation more in depth? (arguments etc)

Reply:
All functions are undocumented. The `react-select` docs offer enough reference IMHO. I don't want to repeat them multiple times (`SelectInput`, `AsyncSelectInput`, `CreatableSelectInput`, `AsyncCreatableSelectInput`, `SelectField`, `AsyncSelectField`, `CreatableSelectField`, `AsyncCreatableSelectField`).

Reply:
So maybe we can point out this documentation in here?

Reply:
Alright. No worries. I just noticed this because I got a bit lost trying to understand it